src/js/index.js

```diff
@@ -1,25 +1,6 @@
 import * as Errors from '#shared/errors';
+import Connection from '../ts/Connection';
 import Document from '../ts/Document';
 import Schema from '../ts/Schema';
-import createConnection from './createConnection';
 
-/**
- * Main mvom module
- * @module mvom
- */
-const mvom = {
-	/**
-	 * @member {Errors} Errors
-	 * @memberof mvom
-	 */
-	Errors,
-	/**
-	 * @member {Schema} Schema
-	 * @memberof mvom
-	 */
-	Schema,
-	createConnection,
-};
-
-export default mvom;
-export { Document, Errors, Schema, createConnection };
+export { Document, Errors, Schema, Connection };
```

Comment:
I changed this export pretty significantly which will potentially be a breaking change when the release is created.  I see no reason to have an default export of an object as well as named exports.
Additionally, `createConnection` was eliminated as a standalone factory function and instead is now a static factory method on `Connection`